motioneye/monitor.py

```diff
@@ -31,7 +32,10 @@
 
 
 def get_monitor_info(camera_id):
-    now = time.time()
+    if sys.platform[:3] == 'win':
+        now = time.clock()
+    else:
+        now = time.time()
```

Comment:
`time.clock()` is deprecated as of Python 3, hence this would need to be reverted:
```suggestion
    now = time.time()
```